README.md

```diff
@@ -9,9 +9,23 @@ It provides a new index type for vector columns called `hnsw` which speeds up `O
 
 ## Quickstart
 
-Note: Currently LanternDB depends on [pgvector](https://github.com/pgvector/pgvector) for the `vector` data type. You'll need to manually install pgvector before moving to the next step.
+### Requirements
 
-LanternDB builds and uses [usearch](https://github.com/unum-cloud/usearch) for its single-header state of the art HNSW implementation.
+Make sure you have the following software installed:
+
+- `gcc` (version >= ?)
+- `g++` (version >= ?)
+- `postgres` (version >= ?)
+- `cmake` (version >= ?)
+- `make` (version >= ?)
+
+#### Installing pgvector
+
+Before proceeding, you need to install [pgvector](https://github.com/pgvector/pgvector), which is a dependency for LanternDB.
```

Comment:
This will soon be unnecessary. no need to add it to the readme.

Reply:
Done pleas have a look @Ngalstyan4